Directory.Build.props

```diff
@@ -12,12 +12,14 @@
 
     <!-- Allow references to unsigned assemblies (like MAUI) from signed projects -->
     <NoWarn>$(NoWarn);CS8002</NoWarn>
+  </PropertyGroup>
 
-    <!-- Configure Sentry CLI -->
+  <!-- Configure Sentry CLI -->
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
     <SentryOrg>sentry-sdks</SentryOrg>
     <SentryProject>sentry-dotnet</SentryProject>
-    <SentryUploadSources Condition="'$(Configuration)' == 'Release'">true</SentryUploadSources>
-
+    <SentryUploadSymbols>true</SentryUploadSymbols>
```

Comment:
```suggestion
    <!-- <SentryAuth>This value is set via environment variable but could be set here</SentryAuth > -->
    <SentryUploadSymbols>true</SentryUploadSymbols>
```

Reply:
How about something like this so it's clear one value is "missing" but is set elsewhere?

Reply:
For us, sure.  Just not for the public recommendation in the corresponding docs.